formats/datatables/SearchPanes.php

```diff
@@ -132,30 +132,31 @@ private function getPanesOptions(
 		if ( $isCategory ) {
 
 			// data-length without the GROUP BY clause
-			$sql_options = [ 'LIMIT' => 1 ];
-
-			$dataLength = (int)$this->connection->selectField(
-				$this->connection->tableName( $qobj->joinTable ) . " AS $qobj->alias" . $qobj->from
-					. ' JOIN ' . $this->connection->tableName( 'smw_fpt_inst' ) . " AS insts ON $qobj->alias.smw_id = insts.s_id",
-				"COUNT(*) AS count",
-				$qobj->where,
-				__METHOD__,
-				$sql_options
+			$sql_options = ' LIMIT 1';
+
+			$res = $this->connection->query(
+				'SELECT COUNT(*) as count' .
+				' FROM ' . $this->connection->tableName( $qobj->joinTable ) . " AS $qobj->alias" . $qobj->from
+					. ' JOIN ' . $this->connection->tableName( 'smw_fpt_inst' ) . " AS insts ON $qobj->alias.smw_id = insts.s_id" .
+				$qobj->where .
+				$sql_options,
+				__METHOD__
 			);
+			$row = $res->fetchRow();
+			$dataLength = (int)( $row['count'] ?? 0 );
 
 			if ( !$dataLength ) {
 				return [];
 			}
 
 			$groupBy = "i.smw_id";
 			$orderBy = "count DESC, $groupBy ASC";
-			$sql_options = [
-				'GROUP BY' => $groupBy,
+			$sql_options =
```

Comment:
Need to verify the old version did not have escaping OR that the new version is fine without it